pybikes/data/cyclehire.json

```diff
@@ -0,0 +1,17 @@
+{
+    "instances": [
+        {
+            "tag": "cyclehire",
```

Comment:
Let's go for the safest and set the tag as `cycle-hire-slough`. The probability of another "Cycle Hire" popping somewhere is high.